.circleci/config.yml

```diff
@@ -347,7 +347,7 @@ jobs:
 
   chk_pylint:
     docker:
-      - image: buildpack-deps:eoan
+      - image: buildpack-deps:focal
```

Comment:
Note to self: We are moving from Ubuntu 19.10 to 20.04 LTS for pylint and antlr CI jobs